docs/src/tutorials/algorithms/parallelism.md

```diff
@@ -205,10 +211,13 @@ set_silent(model)
 @variable(model, x)
 @objective(model, Min, x)
 solutions = Float64[]
+my_lock = Threads.ReentrantLock()
 Threads.@threads for i in 1:10
     set_lower_bound(x, i)
     optimize!(model)
-    push!(solutions, objective_value(model))
+    Threads.lock(my_lock) do
+        push!(solutions, objective_value(model))
```

Comment:
For this one, maybe it makes sense to push `(i, objective_value(model))` or something? Since they might not be in order.

Reply:
:+1: updated